data/ICE_consumer_data.jl

```diff
@@ -0,0 +1,28 @@
+residential = Residential(20, 550, -0.005, nothing,
```

Comment:
All of the content of this file can be copied into `https://github.com/jd-lara/PVCR/blob/master/data/consumer_data.jl` and then just rename the variables.